lib/session-picker.coffee

```diff
@@ -0,0 +1,112 @@
+{SelectListView} = require 'atom-space-pen-views'
```

Comment:
As I commented above, I think the user should choose kernels rather than sessions.

The current workflow is that users choose a ZMQ kernel amongst those that are able to handle the current grammar.

This PR should aim at the same workflow for WS kernels, i.e. the user should be shown a list of WS kernels that are able to handle the current grammar.